src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+  distributed with the SDK and documented at [api.dartlang.org]({{site.dart_api}}).
+* Libraries shared with the Dart community are distributed as library packages,
+  published at [pub.dartlang.org](https://pub.dartlang.org/).
+  The [pub](/tools/pub/) tool allows you to create, publish, and manage library packages.
+* You can use libraries from GitHub or your local file system by placing them in the
+  `/lib` directory of your [application's directory
+  structure](https://www.dartlang.org/tools/pub/package-layout#public-directories).
+
+This document discusses the first two kinds of libraries,
+and tells you where to learn more about some of the most widely used Dart libraries.
+
+<aside class="alert alert-info" markdown="1">
+**Tip:**
+If you don't find the functionality you need on [api.dartlang.org]({{site.dart_api}}),
+check [pub.dartlang.org](https://pub.dartlang.org/).
+</aside>
+
+Looking for web, server, or Flutter libraries?
+See [Specialized libraries](#specialized-libraries).
+
+## Dart SDK libraries
+
+The SDK libraries (dart:core, dart:async, dart:math, dart:convert) contain the
+fundamental classes used in Dart applications.
+Classes that aren't as universal are placed in packages outside of the SDK.
+
+The [library tour](/guides/libraries/library-tour) walks you through the
+libraries distributed with the SDK.
+
+## Useful packages
+
+Developers have written some excellent packages for use by the
+Dart community.  Here are some popular and useful packages,
+in alphabetical order:
+
+| **Package** | **Description** | **Commonly used APIs** |
+| [firebase3](https://pub.dartlang.org/packages/firebase3) | A wrapper for [Firebase](https://firebase.google.com) that allows you to easily publish your app to the cloud. | Auth, Database, Query, Storage |
+| [http](https://pub.dartlang.org/packages/http) | A set of high-level functions and classes that make it easy to consume HTTP resources. | delete(), get(), post(), read() |
+| [intl](https://pub.dartlang.org/packages/intl) | Internationalization and localization facilities, and support for plurals and genders, date and number formatting and parsing, and bidirectional text. | Bidi, DateFormat, MicroMoney, TextDirection |
+| [logging](https://pub.dartlang.org/packages/logging) | A configurable mechanism for adding message logging to your application. | LoggerHandler, Level, LogRecord |
+| [mockito](https://pub.dartlang.org/packages/mockito) | A popular framework for mocking objects in tests. Especially useful if you are writing tests for dependency injection. Used in conjunction with the [test](https://pub.dartlang.org/packages/test) package. | Answering, Expectation, Verification |
+| [path](https://pub.dartlang.org/packages/path) | Common operations for manipulating different types of paths. For more information, [Unboxing Packages: path](http://news.dartlang.org/2016/06/unboxing-packages-path.html). | absolute(), basename(), extension(), join(), normalize(), relative(), split() |
```

Comment:
information, -> information, see

OR
For more information, -> More information: